kraft.yaml

```
@@ -0,0 +1,46 @@
+specification: '0.4'
+
+unikraft:
+  version: 'gcov'
+  kconfig:
+    # Enable 9pfs filesystem support.
+    - CONFIG_LIBUK9P=y
+    - CONFIG_LIB9PFS=y
+    - CONFIG_LIBVFSCORE_AUTOMOUNT_ROOTFS=y
+    - CONFIG_LIBVFSCORE_ROOTFS_9PFS=y
+    - CONFIG_LIBUKLIBPARAM=y
+    # Enable profile-guided optimization (PGO) and coverage support.
+    - CONFIG_OPTIMIZE_PGO=y
+    - CONFIG_OPTIMIZE_PGO_GENERATE=y
```

Add a comment mentioning that this line must be changed to run `use`.
```suggestion
    # Change the line below to *_USE=y
    - CONFIG_OPTIMIZE_PGO_GENERATE=y
```